lib/lint.ml

```diff
@@ -260,6 +260,11 @@ module Checks = struct
         if is_build then [ (pkg, DuneIsBuild) ] else errors
     | None -> []
 
+  let check_tags ~pkg opam =
+    match OpamFile.OPAM.tags opam with
+    | "topics" :: _ -> [ (pkg, TopicsTagPresent) ]
+    | _ -> []
```

Comment:
Could we also expand this so that it ensures the default example tags have either been removed or replaced with substantive topics?